test-module/src/main/java/no/unit/nva/doi/requests/util/DoiRequestsDynamoDBLocal.java

```diff
@@ -82,6 +84,15 @@ protected void insertPublication(String tableName, Publication publication) thro
         );
     }
 
+    protected Publication getPublication(String tableName, UUID publicationId, String modifiedDate) throws IOException {
```

Comment:
method for reading directly the publication for the table. Used for  verifying that the service actually saves stuff in the database